notebooks/wav2lip/wav2lip.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    sys.path.append(str(wav2lip_path))
</pre><p>as we discussed sometime ago, it is better to use insert(0) instead to append for avoiding mixing packages in env and too long variables cutting on windows</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>fixed</p>